core/types/vote.go

```diff
@@ -0,0 +1,89 @@
+package types
+
+import (
+	"sync/atomic"
+
+	"github.com/ethereum/go-ethereum/crypto/bls"
+	"github.com/pkg/errors"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	BLSPublicKeyLength = 48
+	BLSSignatureLength = 96
```

Comment:
These constants are in params/config.go already, so we don't need to redefine them here